constants.ts

```diff
@@ -23,8 +23,8 @@ export const eip165ABI = [
 
 // Sample Address Inputs
 export enum SAMPLE_ADDRESS {
-  MAINNET_UP = '0x29d7c7E4571a83B3eF5C867f75c81D736a9D58aa',
+  MAINNET_UP = '0x0F4180da178ed1C71398a57ca8Cb177F69591f1f',
   TESTNET_UP = '0x027b6f7be4399727d4e0132c2cE027Cd3e015364',
-  MAINNET_LSP7 = '0x',
-  TESTNET_LS7 = '0xD02629cdA51b46408348CE94D1D28200524FFC33',
+  MAINNET_LSP7 = '0x8DA488C29FB873c9561cCF5FF44Dda6C1DedDC37',
+  TESTNET_LS7 = '0x4A1f3bB9806dBE56e6B95e262161Ce176C6C9E3e',
```

Comment:
for some reasons, this address does not show up as LSP7 on testnet

<img width="698" alt="image" src="https://github.com/lukso-network/tools-erc725-inspect/assets/477945/6ced502f-fc10-4629-8f03-83f9a25febac">